doc/src/overrides.md

```diff
@@ -111,7 +111,8 @@ that uses the new TOML encoding in the `rust-toolchain` file. You need to upgrad
 `rustup` to 1.23.0+.
 
 The `rust-toolchain.toml`/`rust-toolchain` files are suitable to check in to
-source control.
+source control. If that's done, `Cargo.lock` should probably be tracked too to
+avoid potential compatibility issues with dependencies.
```

Comment:
This only applies to toolchain files that specify a fixed version - its not true for e.g. 'stable'. Could you please update this to add that nuance, then I think we can merge it.